backend/endpoints/change_password.go

```diff
@@ -0,0 +1,332 @@
+package endpoints
+
+import (
+	"context"
+	"errors"
+	"log"
+	"math"
+	"regexp"
+	"strings"
+
+	"github.com/go-chi/jwtauth"
+	"github.com/go-kit/kit/endpoint"
+
+	jwtAuth "github.com/dwarvesf/smithy/backend/auth"
+	backendConfig "github.com/dwarvesf/smithy/backend/config"
+
+	"github.com/dwarvesf/smithy/backend/service"
+)
+
+// ChangePasswordRequest store change password structer
+type ChangePasswordRequest struct {
+	OldPassword   string `json:"old_password"`
+	NewPassword   string `json:"new_password"`
+	ReNewPassword string `json:"re_new_password"`
+}
+
+// ChangePasswordResponse store change password respone
+type ChangePasswordResponse struct {
+	Complexity string `json:"complexity"`
+}
+
+const (
+	TooShort   = "Too Short"
+	VeryWeak   = "Very Weak"
+	Weak       = "Weak"
+	Good       = "Good"
+	Strong     = "Strong"
+	VeryStrong = "Very Strong"
+	// config name
+	configFilePath = "example_dashboard_config.yaml"
```

Comment:
we should not using constant for `configFilepath` because is not correct in many case (that name can change)